index.html

```diff
@@ -14,25 +14,32 @@
 
 <body>
     <section id="app">
+        <context-menu ref="menu"></context-menu>
         <config-menu></config-menu>
-        <header id="main-header" class="margin-bottom" style="height:100px">
+        <header id="main-header" class="margin-bottom" style="height:100px" v-on:contextmenu.prevent="$refs.menu.openContextMenu">
             <navbar v-bind:suites="suites"></navbar>
         </header>
 
-        <main v-bind:class="{ 'full-size': !AppStatus.config.bottomBar }">
+        <main v-bind:class="{ 'full-size': !AppStatus.config.bottomBar }" v-on:contextmenu.prevent="$refs.menu.openContextMenu">
             <template v-for="(suite,index) in suites">
                     <task-suite v-bind:suite="suite" v-bind:index="index"></task-suite>
                 </template>
         </main>
 
-        <footer v-if="AppStatus.config.bottomBar">
+        <footer v-if="AppStatus.config.bottomBar" v-on:contextmenu.prevent="$refs.menu.openContextMenu">
             <bottom-bar></bottom-bar>
         </footer>
     </section>
 
     <script>
         window.$ = window.jQuery = require('./node_modules/jquery/dist/jquery.min.js');
         window.Hammer = require('./node_modules/materialize-css/js/hammer.min.js');
+        window.document.body.addEventListener('click', function (ev) {
```

Comment:
do we need this hardcoded in the html?

Reply:
No I think that could probably go into app.js

Reply:
Ok, please, put it in app.js if possible and I think that's all, thanks